src/assets/stylesheets/application.scss

```diff
@@ -56,6 +56,8 @@
 @import "extensions/footnotes";
 @import "extensions/permalinks";
 
+@import "clipboardjs/clipboardjs";
```

Comment:
Cool, this is what I was looking for.  I wasn't sure what to call it, but I figured you wanted to shove it down somewhere.